api/models/reservation.py

```diff
@@ -0,0 +1,31 @@
+"""Payment model."""
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from .base import Base
+from .order import Order
+from .workshopPrice import WorkshopPrice
+
+
+class Reservation(Base):
+    """Stores workshop reservations."""
+
+    workshop_price = models.ForeignKey(
+        WorkshopPrice,
+        verbose_name=_("Workshop price"),
+    )
+    order = models.ForeignKey(
+        Order,
+        verbose_name=_("Order"),
+    )
+    ends_at = models.DateTimeField(
```

Comment:
It seems that every second Django convention has, is just alien to me. Final decision is on you, but I suggest NOT using conventions that:

* will bring you extra work
* are not necessary, nor justified
* hurt my eyes

Btw, you can't translate underscore to CamelCase, their relationship is not one to one.